ios/Podfile

```diff
@@ -1,4 +1,4 @@
-source 'https://github.com/CocoaPods/Specs.git'
+source 'https://cdn.cocoapods.org/'
```

Comment:
this was temp;orary fix because CDN was down.
you could remove this line totally

Reply:
ok